src/Microsoft.VisualBasic/pkg/Microsoft.VisualBasic.builds

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <!-- Temporary workaround, VisualBasic projects cannot be built on non-Windows platforms. See issue #5230. -->
```

Comment:
Yes please remove the comment.

cc @ericstj who is going to be working on the individual library packages.